src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -115,55 +40,115 @@ const ChatItemMyTextV2 = ({
   message = '',
   status = ChatStatus.PENDING,
   avatar,
-  chatType
+  chatType,
+  messageType,
+  data
 }: ChatItemMyTextProps) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ChatItemMyTextProps` type should be updated to include the new properties `chatType`, `messageType`, and `data`. If these properties are optional, they should have default values or null checks to prevent potential runtime errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->